src/plugins/renamer.ts

```diff
@@ -0,0 +1,32 @@
+import { CodeModel, codeModelSchema, Language, Parameter } from "@azure-tools/codemodel";
+import { Session, startSession, Host, Channel } from "@azure-tools/autorest-extension-base";
+import { serialize, deserialize } from "@azure-tools/codegen";
+
+export class Renamer {
+    codeModel: CodeModel;
+
+    constructor(protected session: Session<CodeModel>) {
+        this.codeModel = session.model;
+    }
+
+    async process() {
+        return this.codeModel;
+    }
+}
+
+export async function processRequest(host: Host) {
+    const debug = await host.GetValue('debug') || false;
+
+    try {
+        const session = await startSession<CodeModel>(host, {}, codeModelSchema);
+        const plugin = new Renamer(session);
+        const result = await plugin.process();
+        host.WriteFile('code-model-cli-v4.yaml', serialize(result));
```

Comment:
>code-model-cli-v4.yaml [](start = 24, length = 22)

Add to PathConstants in model.ts

Reply:
Can we change the name of model.ts. we already has a modeler.ts. it's prettty confusing.